m4_sessions/1811_inning/failure.md

```diff
@@ -25,6 +29,26 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Introductions, name, mod, something you are good at and how long it took you to actually become good at it 5 mins
+
+Short video  talking about how failure is a good thing 5 mins
+
+Redefinition of failure 7 mins. (ask how people would define failure, ask how people feel when they fail, ask what they think of others when they fail)
+
+Short video of successful people and their failures 5 min
+
+Discussion about how failure is not normalized and how if we knew about every time people had failed rather than only the successful stories that people would not feel so bad about it. 7min
```

Comment:
Think about how to manage the flow of the discussion--how big are the groups?  Are you each in one of the groups?

Reply:
https://www.youtube.com/watch?v=zLYECIjmnQs

video of successful people and their failures